Mathlib/Logic/Equiv/Fin.lean

```diff
@@ -498,3 +498,13 @@ instance subsingleton_fin_zero : Subsingleton (Fin 0) :=
 /-- `Fin 1` is a subsingleton. -/
 instance subsingleton_fin_one : Subsingleton (Fin 1) :=
   finOneEquiv.subsingleton
+
+/-- The natural `Equiv` between `(Fin m → α) × (Fin n → α)` and `(Fin (m + n) → α)`.-/
+def Equiv.finArrowProdEquivFinAddArrow {α : Type*} (m n : ℕ) :
```

Comment:
It's super helpful to put 
```suggestion
@[simps]
def Equiv.finArrowProdEquivFinAddArrow {α : Type*} (m n : ℕ) :
```
on definitions like these, as then Lean generates a `@[simp]` (no `s`) lemma explaining what the forward and reverse directions of the map are.